src/client/components/ProductsPagination/ProductsPaginationImages.component.js

```diff
@@ -0,0 +1,27 @@
+import image01 from '../../assets/images/image01.png';
```

Comment:
These images are not exclusive from Pagination. Can't you find a better logic for this?

Reply:
Hi @dpfernandes i tried to access the images on better ways, also i discussed with mentors from Zendesk & also my classmates, but for now i could not find any better solution. But when we have database is ready with the seeds, then may be we have better approach to access them directly from database rather than from another component , which is now iam doing. But still iam researching better approach .....thanks for pushing me

Reply:
@BikramRawat  I think the best way if your Component takes an array of img links in props. Make sure to add proptypes and default placeholder.